### PR TITLE
[11.0][3625][ADD] account_security_adj

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3625](https://www.quartile.co/web#id=3625&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: account_security_adj/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Account Security Adj",
+    "version": "11.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Accounting",
+    "license": "LGPL-3",
```
Comment (yostashiro) [inline account_security_adj/__manifest__.py]:
```suggestion
    "license": "AGPL-3",
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please add CONTEXT.rst, mentioning that the adjustments of this module is not required in newer versions.

Please add the setup file.

**Comment by AungKoKoLin1997** [comment]
> Please add CONTEXT.rst, mentioning that the adjustments of this module is not required in newer versions.

Sorry. I don't get the point. Why this adjustment is not required in newer versions? `confirm` button is allowed to group_account_invoice in newer version.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 My bad. Please forget about the context file.

File: account_security_adj/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module restricts the validation of customer invoices and vendor bills to the group_account_manager group.
```
Comment (yostashiro) [inline account_security_adj/readme/DESCRIPTION.rst:1]: @kanda999 Can you update this to reflect the latest design in concise manner.

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]

**Pull request merged**